packages/engine/bin/cli/src/main.rs

```diff
@@ -1,9 +1,9 @@
-//! The hEngine CLI
+//! The HASH Engine CLI
 //!
-//! A binary responsible for the orchestration and management of hEngine processes that are used to
-//! run HASH simulation projects. This CLI is a light-weight implementation of an [`orchestrator`]
-//! which enables the running of an experiment through the command-line, accepting a variety of
-//! [`Args`].
+//! A binary responsible for the orchestration and management of HASH Engine (hEngine)
+//! processes that are used to run HASH simulation projects. This CLI is a light-weight
+//! implementation of an [`orchestrator`] which enables the running of an experiment
+//! through the command-line, accepting a variety of [`Args`].
```

Comment:
We use a column witdh of 100 in Rust
```suggestion
//! A binary responsible for the orchestration and management of HASH Engine (hEngine) processes
//! that are used to run HASH simulation projects. This CLI is a light-weight implementation of an
//! [`orchestrator`] which enables the running of an experiment through the command-line, accepting
//! a variety of [`Args`].
```